spec/development/workload/OneNote.ts

```diff
@@ -85,6 +85,22 @@ describe("OneNote", function() {
 		}
 	});
 
+	it("Create a OneNote page with application/xhtml+xml page content", async () => {
+		try {
```

Comment:
Please remove the try catch block if you are going to rethrow the same exact error in the catch block without additional handling.

Reply:
Removed the case. There are more tests in the project which are using try catch and throwing the error. I will clean that up in a separate PR.